### ISSUE TITLE
Landscape Screen Orientation does not work on 2116

### ISSUE THREAD

**Issue by Nate0714** [Lime3DS/Lime-3DS-Emulator#348]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Lime3ds 2116

### Description of Issue

Selecting in app options for changing Landscape Screen Orientation does not work. Single screen, hybrid, side by side etc. are non functional options on 2116 update including when hotkeyed. Only workarounds are selecting a new orientation and closing the app, or having rotate screen on in device and physically rotating the device and selecting a new orientation. The device I have is Retroid Pocket 2s running android 11 lime 3ds 2116

### Expected Behavior

The expected behavior is selecting a new option like single screen will show just one screen however instead the screens revert to two screens vertically oriented and only swapping screens has any effect. Any change to Landscape Screen Orientation does not work or do anything to change the screens being vertically placed one on top of the other. An example of this working correctly is melon ds or citra where toggling a new layout like single screen will display one screen on the device.

### Reproduction Steps

Open lime3ds on latest update 2116, open the side bar in app and choose a new option for Landscape Screen Orientation and visualize no change to the screen layout, if you exited the app and reopened to force the change watch as it reverts to two screens vertically displayed with no further adjustments having any effect.  This includes hitting the hotkey you mapped this function to. From this vertical layout only swap screens will function still to change which is on top or bottom and vice versa. The only way to effect a change in the layout from this point is to select the desired layout (like single screen) and exit the app and reopen or physically rotate the device and select the new choice again to have the change take effect.

### Log File

[lime3ds_log.txt](https://github.com/user-attachments/files/16578342/lime3ds_log.txt)
(I've attached the log both in an edit of the ticket and as a comment, thank you for taking the time to look into this)

### System Configuration

CPU: 2*A75@1.8G/6*A55@1.8G
GPU/Driver: Mali G52MC2@614MHz 

https://github.com/user-attachments/assets/1eb6d9a5-a3de-42e5-b9d3-07dc86f7e07f

RAM: 4gb
OS:android 11

**Comment by OpenSauce04** [comment]
Please attach a log file as required by the issue template

**Comment by Nate0714** [comment]
[lime3ds_log.txt](https://github.com/user-attachments/files/16578319/lime3ds_log.txt)


This log was much larger, but when I tried to replicate the screen orientation error and physically rotating my device to restore functionality to the screen it crashed several times and the log was reduced from almost 44 mb down to this few kb log.  I can replicate the error more to build up the log file again if needed.

**Comment by OpenSauce04** [comment]
The crash in the log appears to be unrelated to the original issue and seems to be caused by a bad ROM

**Comment by OpenSauce04** [comment]
The crash in the log appears to be unrelated to the original issue and seems to be caused by a bad ROM

**Comment by Nate0714** [comment]
I can still replicate the crash using a new ROM that I did not decrypt myself and the md5 check passes, however I think that is a smaller issue as these crashes only occur when exiting the app and reopening to fix the primary issue of adjusting the screen layout.  This crash can also occur by toggling a new layout, and then pausing emulation and resuming to have the new orientation take effect.  Here are two additional log files for your review.  I have about 40 hours on the old Rom the original log was pertaining to and 1 hour on this new one, it runs the same and can have the crashes occur by doing the same steps.  Thank you for looking into this.
[lime3ds_log(1).txt](https://github.com/user-attachments/files/16592459/lime3ds_log.1.txt)
[lime3ds_log.old.zip](https://github.com/user-attachments/files/16592461/lime3ds_log.old.zip)
I had to zip the old log that was created from yesterday to today as it is around 45 MB and git limits to 20.  Thank you

**Comment by OpenSauce04** [comment]
That's strange, the log seems to be blatantly complaining about missing files in the ROM unless I'm misinterpreting it really badly.

Does this happen with other games?

**Comment by Nate0714** [comment]
I see what you mean with the errors showing it can't locate the files, however I am able to play all 4 of the games i currently have, and I now have 2 Roms for mh3u with both being functional.  I can post another video if needed.  They all work fine as far as I can tell although I have only gotten through about half of mh3u.  The only thing I can think of is I changed the location of these files recently when I was trying to see if maybe moving them to the internal storage itself rather than the sd card I usually keep roms on fixes the screen issue and maybe increase performance.  I have since moved them back but maybe the app is still trying to view the original location they were at.  This app works flawlessly either way, really only the screen orientation is the only thing that is weird and even then I can correct it by just physically getting the screen to rotate which updates the layout.  I only replicate the crash doing the original solution I used before finding out about the rotate screen, in which i pause the emulation and resume to update the layout orientation or close the app and reopen several times as I adjust the layout and see if it updated.  Outside of these two scenarios the app runs without crashing.  Thank you!
